src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -1691,7 +1692,13 @@ public static void selectProjectFromRemoveLibertyProjectDialog(RemoteRobot remot
      * @param waitTime    The time (seconds) to wait for the required message to appear in the text area.
      */
     public static void waitForLTWNoProjectDetectedMsg(RemoteRobot remoteRobot, int waitTime) {
-        String text = "No Liberty Maven or Liberty Gradle projects detected in this workspace.";
+        String text = " 1. If no projects are open in the Intellij Explorer, open or create a Liberty project using the" +
+                " File menu.  2. If there is one or more existing Maven or Gradle projects open in the Intellij " +
+                "Explorer, try one of the following:   a. Configure the Liberty build plugin in the build file of an " +
+                "existing Maven or Gradle project. b. Add a server.xml file to an existing Maven or Gradle project" +
+                " at 'src/main/liberty/config'. c. Manually add an existing Maven or Gradle project to the Liberty " +
+                "Dashboard through the Intellij Explorer command palette action 'Liberty:'Add project to Liberty " +
+                "Dashboard'";
```

Comment:
please update the test case message to match

Reply:
should the text here match what is in LibertyBundles.properties? It looks ok on its own but im not sure based on your comment if it's still awaiting changes to be like the other file

Reply:
@dmuelle Yes, the text should be updated here as well. If you are good with the text as it is in LibertyBundles.properties, feel free to approve. No need to wait for the other changes to come in, they are just for testing. I will make sure that gets handled.